web_src/css/helpers.css

```diff
@@ -158,6 +158,12 @@ Gitea's private styles use `g-` prefix.
 .gt-mt-4 { margin-top: 1rem !important; }
 .gt-mt-5 { margin-top: 2rem !important; }
 
+.-gt-mt-1 { margin-top: -.125rem !important; }
+.-gt-mt-2 { margin-top: -.25rem !important; }
+.-gt-mt-3 { margin-top: -.5rem !important; }
+.-gt-mt-4 { margin-top: -1rem !important; }
+.-gt-mt-5 { margin-top: -2rem !important; }
```

Comment:
It doesn't seem to be tailwind style.

ps: I also prefer to use something else, instead of tricky negative margin -- if possible.

![image](https://github.com/go-gitea/gitea/assets/2114189/f92c284f-b5db-4b57-8d24-ce6bccc8e847)

Reply:
@wxiaoguang My point was just that we seem to be following tailwind naming conventions, but with `gt-*` prefixed. In tailwind negative margin would be `-mt-4`. Are you suggesting `gt--mt-2` instead of `-gt-mt-2`?

I can use position instead of margin to remove a negative. I think it's less intuitive, but something like [this would work](https://play.tailwindcss.com/Vt1urArWoZ) if you like it better. I'd just need to generate the top/bottom position helper class definitions. 

BTW, why are we recreating tailwind? Is it because there are too many class conflicts?

Reply:
> So are you suggesting `gt--mt-2` instead of `-gt-mt-2`?

See the screenshot, I think it is `gt--mt-2` from the official example.

----

I haven't checked the details carefully and have no idea about the what's the "best" approach at the moment.

Reply:
oh, you're saying we follow the `tw-*` [prefix syntax](https://tailwindcss.com/docs/configuration#prefix). I see... gross, but yeah I agree it's probably the best option given it more closely follows the tailing docs.

Reply:
👍

Reply:
Tailwind's [prefix](https://tailwindcss.com/docs/configuration#prefix) link explicitly points out that it goes _before_ the prefix.

> The dash modifier for negative values should be added before your prefix, so -mt-8 would become -tw-mt-8 if you’ve configured tw- as your prefix:
>
> ```
> <div class="-tw-mt-8">
>   <!-- -->
> </div>
> ```

Also, in the [Tailwind Play screenshot](https://play.tailwindcss.com/zypvTUxU65) above, if you change `tw--bottom-4` to `-tw-bottom-4`, it behaves identically. Apparently it supports both.

Reply:
Good point. I changed it back. This PR should be all set for review.